contracts/Vault.sol

```diff
@@ -668,9 +668,6 @@ contract Vault is
         uint256 idsLen = _ids.length;
 
         for (uint256 i; i < idsLen; ++i) {
-            if (_amounts[i] > deposits[_ids[i]].amount)
-                revert VaultAmountTooLarge();
-
```

Comment:
This is removed because this code was not tested by any test and the same check is present in the function _withdrawSingle on line 930 (033 on the main branch) 

- [if (_deposit.amount < _amount)
            revert VaultCannotWithdrawMoreThanAvailable(); ]

Reply:
well done 👍